### PR TITLE
[Migration Tools] Tests for migration tools backend

### PR DESCRIPTION
**Pull request by tdermendjiev**
Implements #380 

As part of the xsk-ide-migration project, migration tools backend is cloned into ide-migration module's resources. XSKMigrationJSServiceTest is created to run the e2e migration test in [migrate.js](https://github.com/dirigiblelabs/xsk-ide-migration/blob/main/ide-migration/server/test/migrate.js).

### CODE REVIEW

File: releng/server/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -167,11 +169,13 @@
 	        <url-pattern>/services/v3/wiki/*</url-pattern>
 	        <url-pattern>/services/v3/command/*</url-pattern>
 	        <url-pattern>/services/v3/xsk/*</url-pattern>
+          <url-pattern>/services/v3/migration-operations/*</url-pattern>
```
Comment (ThuF) [inline releng/server/src/main/webapp/WEB-INF/web.xml:172]: Not needed?

File: releng/server/src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -167,11 +169,13 @@
...
 	        <url-pattern>/services/v4/web/*</url-pattern>
 	        <url-pattern>/services/v4/js/*</url-pattern>
 	        <url-pattern>/services/v4/wiki/*</url-pattern>
 	        <url-pattern>/services/v4/command/*</url-pattern>
 	        <url-pattern>/services/v4/xsk/*</url-pattern>
+          <url-pattern>/services/v4/migration-operations/*</url-pattern>
```
Comment (ThuF) [inline releng/server/src/main/webapp/WEB-INF/web.xml:178]: Not needed?

File: releng/server/src/main/webapp/WEB-INF/web.xml [hunk 3]
```diff
@@ -186,11 +190,13 @@
 	        <url-pattern>/public/v3/wiki/*</url-pattern>
 	        <url-pattern>/public/v3/command/*</url-pattern>
 	        <url-pattern>/public/v3/xsk/*</url-pattern>
+          <url-pattern>/public/v3/migration-operations/*</url-pattern>
```
Comment (ThuF) [inline releng/server/src/main/webapp/WEB-INF/web.xml:193]: Not needed?

File: releng/server/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -186,11 +190,13 @@
...
 	        <url-pattern>/public/v4/web/*</url-pattern>
 	        <url-pattern>/public/v4/js/*</url-pattern>
 	        <url-pattern>/public/v4/wiki/*</url-pattern>
 	        <url-pattern>/public/v4/command/*</url-pattern>
 	        <url-pattern>/public/v4/xsk/*</url-pattern>
+          <url-pattern>/public/v4/migration-operations/*</url-pattern>
```
Comment (ThuF) [inline releng/server/src/main/webapp/WEB-INF/web.xml:199]: Not needed?

File: modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-launch-view.js [hunk 5]
```diff
@@ -9,14 +9,15 @@
...
+import static org.junit.Assert.assertTrue;
+
+import org.graalvm.polyglot.HostAccess;
+
+public class AssertHolder {
+  @HostAccess.Export
+  public void callAssertTrue(Boolean res) {
+      assertTrue(res);
+  }
+}
```
Comment (ThuF) [inline modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-launch-view.js]: I really don't get this change

File: modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-perspective-view.js [hunk 6]
```diff
@@ -9,12 +9,12 @@
...
\ No newline at end of file
+package com.sap.xsk.migration.neo.db.hana.connectivity;
+
+public interface HanaConnector  {
+
+  HanaConnectorRes connect(HanaConnectorArgs connectorArgs);
+
+  void disconnect(String connectionId);
+
+}
```
Comment (ThuF) [inline modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-perspective-view.js]: I really don't get this change

**Review by ThuF — CHANGES_REQUESTED** [review]
Remove the following folders as they are not needed:
- `neo-sdk/javadoc`
- `neo-sdk/samples`
- `neo-sdk/tools/samples`

File: modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-launch-view.js (hunk 5 above)
Comment (ThuF) [reply modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-launch-view.js]: Maybe try to build the module/whole project with:
- `mvn clean install -P content` - to fetch the content from the xsk-ide-migration project in DirigibleLabs
- `mvn clean install` - to update the license headers

File: modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-perspective-view.js (hunk 6 above)
Comment (ThuF) [reply modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-perspective-view.js]: Maybe try to build the module/whole project with:
- `mvn clean install -P content` - to fetch the content from the xsk-ide-migration project in DirigibleLabs
- `mvn clean install` - to update the license headers

**Comment by tdermendjiev** [comment]
Closing in favor of #438

**Pull request closed without merging**